notebooks/working/5.0-mab-simulation-causal-graph.md

```diff
@@ -0,0 +1,1432 @@
+                                                                                   size = n_samples),
+    "household_size": lambda n_samples: bike_data_no_parent['household_size'],
+    "num_kids": lambda n_samples: bike_data_no_parent['num_kids'],
+    "num_licensed_drivers": lambda n_samples: bike_data_no_parent['num_licensed_drivers'],
+    "num_cars": lambda n_samples: bike_data_no_parent['num_cars'],
+    "cross_bay": lambda n_samples: bike_data_no_parent['cross_bay'],
+    "utility_shared_3+": lambda n_samples: np.zeros(shape=n_samples)
+})
+<!-- #endraw -->
+
+### Walk-Transit-Walk
+
+```python
+V_WTW.draw()
+```
+
+```python
+bike_data_wide['total_travel_cost_wtw'] = WTW_Reg['total_travel_cost_on_total_travel_time'].params[0] +\
+                                                  WTW_Reg['total_travel_cost_on_total_travel_time'].params[1] *\
+                                                  bike_data_no_parent['total_travel_time_wtw'] +\
+                                                  np.random.normal(loc=0,
+                                                                   scale = WTW_Reg['total_travel_cost_on_total_travel_time'].resid.std(),
+                                                                   size = 4000)
+```
+
+### Drive-Transit-Walk
+
+```python
+V_DTW.draw()
+```
+
+```python
+bike_data_wide['total_travel_cost_dtw'] = DTW_Reg['total_travel_cost_on_total_travel_time'].params[0] +\
+                                                  DTW_Reg['total_travel_cost_on_total_travel_time'].params[1] *\
+                                                  bike_data_no_parent['total_travel_time_dtw'] +\
+                                                  np.random.normal(loc=0,
+                                                                   scale = DTW_Reg['total_travel_cost_on_total_travel_time'].resid.std(),
+                                                                   size = 4000)
+```
+
+### Walk-Transit-Drive
+
+```python
+V_WTD.draw()
+```
+
+```python
+bike_data_wide['total_travel_cost_wtd'] = WTD_Reg['total_travel_cost_on_total_travel_time'].params[0] +\
+                                                  WTD_Reg['total_travel_cost_on_total_travel_time'].params[1] *\
+                                                  bike_data_no_parent['total_travel_time_wtd'] +\
+                                                  np.random.normal(loc=0,
+                                                                   scale = WTD_Reg['total_travel_cost_on_total_travel_time'].resid.std(),
+                                                                   size = 4000)
+```
+
+### Walk
+
+```python
+V_Walk.draw()
+```
+
+No need to simulate any other variables since Walk Total Travel Distance was already simulated
+
+
+### Bike
+
+```python
+V_Bike.draw()
+```
+
+No need to simulate any other variables since Bike Total Travel Distance was already simulated
+
+
```

Comment:
@timothyb0912 here is where I simulate data for nodes with parents.

Reply:
ah got it got it!

I was ging through the commits one by one and was on `98f2b`. I left a comment there but wasn't sure if it would show as a comment on the specific commit (which didn't have this yet) or as a comment on the overall file.

It seems it was the latter!